.conan-recipes/llnl-units/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
```

Comment:
Tested locally. Running CMake with this branch removed the cached `units` installation and installed the new one. Changing compiler version in CMake appears to propagate to `units`. And conan installs multiple packages side by side. Installing an older version of scipp did not re-install `units`. This is fine as `units`'s API seems to be backwards compatible.
Removing the installed package and replacing it with a different version can of course break builds but not installed scipp packages because we link against `units` statically.

Given these observations, I think we have to make sure to use a unique version for each `units` package.

Reply:
That is, increment version every time we update the commit it points to?

Reply:
Either that or use a `dev` suffix to the version. If we pin an exact version in `conanfile.txt`, we do not need to have orderable version numbers.